chromium_src/ui/views/view.h

```diff
@@ -0,0 +1,20 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_UI_VIEWS_VIEW_H_
+#define BRAVE_CHROMIUM_SRC_UI_VIEWS_VIEW_H_
+
+class SharedPinnedTabDummyView;
+class SplitViewLocationBar;
+
+#define BRAVE_VIEW_OWNED_BY_CLIENT_PASS_KEY \
+  friend class ::SharedPinnedTabDummyView;  \
```

Comment:
this needs the same `DO NOT ADD TO THIS LIST!` warning from the upstream class and we should also add codeowner here. You can put me down as codeowner for now

Reply:
@simonhong we need to look at what changes we need to remove these from the list

Reply:
Done in https://github.com/brave/brave-core/pull/28471/commits/b338942ed7eb7983c1989c1d4be1025100edab88

Reply:
Filed https://github.com/brave/brave-browser/issues/45957